### PR TITLE
feat(kubernetes): Support for tolerations

### PR DESCRIPTION
**Pull request by rverma-nikiai**
This is required if we want to deploy some spinnaker components on a specific node.

resolves https://github.com/spinnaker/spinnaker/issues/3541

### CODE REVIEW

**Comment by rverma-nikiai** [comment]
@ezimanyi / @plumpy Can you please review this pr, its a very small PR but we are looking forward to get it merged soon :)

File: halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Toleration.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 Snap, Inc.
```
Comment (ezimanyi) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Toleration.java]: This should reference you or your company

File: halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Toleration.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.config.model.v1.node;
+
+import lombok.Data;
+
+@Data
+public class Toleration {
+  String key;
```
Comment (plumpy) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Toleration.java]: could you make these private?

**Pull request merged**